pkgs/applications/networking/sync/onedrive/default.nix

```diff
@@ -7,14 +7,14 @@
 }:
 
 stdenv.mkDerivation rec {
-  pname = "onedrive";
-  version = "2.3.10";
+  pname = "onedrive-abraunegg";
```

Comment:
I would revert this change, it's so that the bot can pick up the right package in repology

https://repology.org/projects/?search=onedrive&maintainer=&category=&inrepo=&notinrepo=&repos=&families=&repos_newest=&families_newest=

Reply:
I too tend to agree with this opinion. Nothing makes `@skilion`'s version superior besides the fact it was there from before. OTH, what makes `@abraunegg`'s version superior IMO is the fact it is actually maintained - `@skilion`'s version was last updated in 2018.

@SRGOM Is this crucial for repology's updaters to pick it up from Nixpkgs? Even if the answer is yes, I don't think it would matter that much because I don't think other repos update this package that often - meaning @ryantm's bot wouldn't try to update it.

BTW if you are interested in making packages updates more autonomic and with better connection to upstream, you may be interested in: https://github.com/repology/repology-updater/issues/938

Reply:
I'm assuming it goes both ways, there's only 1 package from AUR which has onedrive-abraunegg, while we can get consensus from 8 other package managers by keeping it `onedrive`